semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -165,6 +195,16 @@ For example, `x` in `def identity[T](x: T): T` may be modelled by local symbols
 the underlying document. The same logic applies to the type parameter `T`,
 which is also a local definition.
 
+Local symbols must be unique within the underlying document, but they don't have
+to be unique across multiple documents. For example, at the time of writing
+`semanticdb-scalac` produces local symbols named `local0`, `local1`, etc,
+with the counter resetting to zero for every new document.
```

Comment:
Above you suffix with the file.

Reply:
The specification describes the contents of protobuf payloads (the so-called "low-level semantic API"). High-level APIs are free to apply custom transformations to these payloads.